frontend/src/metabase-types/store/admin.ts

```diff
@@ -21,6 +21,7 @@ export interface AdminState {
   permissions: {
     dataPermissions: GroupsPermissions;
     originalDataPermissions: GroupsPermissions;
+    saveError?: string;
```

Comment:
We were referring this element in `frontend/src/metabase/admin/permissions/components/PermissionsPageLayout/PermissionsPageLayout.jsx` and the test wasn't passing, so added this here